### ISSUE TITLE
Expected a value of type 'bool', but got one of type '_JsonMap'

### ISSUE THREAD

**Issue by lucasdidur** [Kerego/tmdb_dart#3]
```
EXCEPTION: Expected a value of type 'bool', but got one of type '_JsonMap'
STACKTRACE: 
dart:sdk_internal                                                                             dtest
package:tmdb_dart/src/models/packages/tmdb_dart/src/models/movie.dart 22:5                    fromJson
package:tmdb_dart/src/models/packages/tmdb_dart/src/services/movie-service.dart 64:21         getMovie
package:stack_trace                                                                           <fn>
package:angular/src/core/change_detection/packages/angular/src/core/zone/ng_zone.dart 144:18  <fn>
dart:sdk_internal                                                                             runUnary
package:angular/src/core/change_detection/packages/angular/src/core/zone/ng_zone.dart 141:18  [_runUnary]
dart:sdk_internal                                                                             <fn>
package:angular/src/core/change_detection/packages/angular/src/core/zone/ng_zone.dart 117:11  <fn>
package:stack_trace                                                                           <fn>
package:angular/src/core/change_detection/packages/angular/src/core/zone/ng_zone.dart 132:18  <fn>
dart:sdk_internal                                                                             run
package:angular/src/core/change_detection/packages/angular/src/core/zone/ng_zone.dart 129:18  [_run]
```

**Comment by Kerego** [comment]
Fixed in 0.0.5

**Comment by lucasdidur** [comment]
Also, to work witch last dart version, the AssetResolver class on the  getPath methods need to return a String, not dynamic
